### DISCUSSION TITLE
Community driven landspaces

### DISCUSSION THREAD

**Discussion by nikolaydubina** [cncf/landscape2-sites#71] [Ideas]
Hi, I come from hacker news where I saw ML infra landscape: https://ai-infra.fun

Is there interest to add list of community landscapes here? I think it would benefit community and this project!

**Comment by tegioz** [comment]
Hi @nikolaydubina 👋

Yes, we'd love to do that, but we'd rather have an ADOPTERS file in the `cncf/landscape2` repository, the landscape generating software. In this repository we maintain the settings and some workflows to build and deploy some landscapes maintained by the Linux Foundation. We'd add these ones to the ADOPTERS file as well, but it's probably better if we list the ADOPTERS where the software lives.

Thanks!
